src/hub/routes/abstract.py

```diff
@@ -17,16 +17,29 @@ def __init__(self, payload) -> None:
 
     def report_route(self, payload: dict, sent_system: str) -> None:
         logger.info("report route", payload=payload, sent_system=sent_system)
-        existing = flask.g.hub_table.get_event(payload["event_id"])
+        event_id = None
+        if payload.get("event_id"):
```

Comment:
why do we have to have this ugly if block for `event_id` and `eventId`?

Reply:
depending on the event type either event_id or eventId may be sent, this is needed to support FxA and Basket.